cms/templates/howitworks.html

```diff
@@ -14,11 +16,10 @@
 <div class="wrapper-content-header wrapper">
   <section class="content content-header">
     <header>
-      <h1><span class="wrapper-text-welcome">${_("Welcome to {studio_name}").format(
```

Comment:
This original string is better for i18n, because `studio_name` could theoretically translate to multiple locations in the string.  I would use `Text()` and `HTML()`.

I'm not great with HTML.  If you can nest these two `spans` and properly override the class for `studio_name`, then I'd recommend you wrap the whole thing with the first span and only include `<span class="logo">` in the translated string with `span_start` and `span_end` arguments to `format()`.  Otherwise, you'll need `welcome_span_start`, `welcome_span_end`, `studio_span_start`, and `studio_span_end`.